### PR TITLE
Doc page Slider component

### PR DESCRIPTION
**Pull request by Jialecl**

### CODE REVIEW

File: website/screens/components/slider/usage/SliderUsagePage.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            As more information can give it to the user, the easier the selection will be.
+          </DxcText>
+          <DxcText>
+            If the value has to be specific, give some resource to the user to fill a precise input,
+            i.e. an input next to the slider.
+          </DxcText>
+        </DxcList>
+      </DxcStack>
+      <DxcStack gutter="large">
+        <HeadingLink level={3}>Slider with input</HeadingLink>
```
Comment (marcialfps) [inline website/screens/components/slider/usage/SliderUsagePage.tsx:28]: We should keep the order of the sections we have in the guidelines: first `Variants`, then `Slider with input`..

File: website/screens/components/slider/specs/SliderSpecsPage.tsx [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+import sliderAnatomy from "./images/slider_anatomy.png";
+import sliderSpecs from "./images/slider_specs.png";
+import sliderStates from "./images/slider_states.png";
+import { DxcLink , DxcList, DxcStack, DxcTable, DxcText } from "@dxc-technology/halstack-react";
+import DocFooter from "../../../common/DocFooter";
+import HeadingLink from "../../../common/HeadingLink";
+import Figure from "../../../common/Figure";
+import Code from "../../../common/Code";
+
+const ButtonSpecsPage = () => {
```
Comment (marcialfps) [inline website/screens/components/slider/specs/SliderSpecsPage.tsx:11]: Change to `SliderSpecsPage`.

File: website/screens/components/slider/specs/SliderSpecsPage.tsx [hunk 3]
```diff
@@ -0,0 +1,704 @@
...
+                <Code>width</Code>
+              </td>
+              <td>Tick</td>
+              <td>-</td>
+              <td>4px</td>
+            </tr>
+          </tbody>
+        </DxcTable>
+        <DxcText as="p">
+          [*] The thumb element size is 14x14px in the following states: <Code>:hover</Code>, and{" "}
```
Comment (marcialfps) [inline website/screens/components/slider/specs/SliderSpecsPage.tsx:561]: Remove comma before `and`.

**Review by marcialfps — CHANGES_REQUESTED** [review]

File: website/screens/components/slider/usage/SliderUsagePage.tsx (hunk 1 above)
Comment (Jialecl) [reply website/screens/components/slider/usage/SliderUsagePage.tsx:28]: That was my first thought too, but the text in Slider with input makes more sense in this position. I was thinking of having the section in this position until the text is changed.

Comment (marcialfps) [reply website/screens/components/slider/usage/SliderUsagePage.tsx:28]: I said that because it refers to the Slider variants. What do you think @aweell ?

Comment (aweell) [reply website/screens/components/slider/usage/SliderUsagePage.tsx:28]: Can you elaborate on "makes more sense" @Jialecl?

I agree with @marcialfps that we should keep the order of the sections given, then we can even discuss if we should call variants the `discrete` and `continuous` forms of the slider.

Comment (Jialecl) [reply website/screens/components/slider/usage/SliderUsagePage.tsx:28]: Based on the text I made the correlation to the usage in which you needed a specific value and also it seems to be introducing the variants.

Comment (aweell) [reply website/screens/components/slider/usage/SliderUsagePage.tsx:28]: I've just read the text and makes sense what you propose for how the section paragraph is redacted (Or copied & pasted from the previous documentation 👎🏼 ).

But still, just to be consistent between components, keep the sections order as:
* Usage 
* Variants
* XYZ...

And I note in the doc issue that the paragraph needs to be re-redacted

**Review by marcialfps — APPROVED** [review]

**Pull request merged**